.github/workflows/linters.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: Linters
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    branches:
+      - main
+      - stable-*
+    tags:
```

Comment:
Is this needed?

Reply:
it is not mandatory but adding it as an extra check